### ISSUE TITLE
feat: link conference notes

### ISSUE THREAD

**Issue by alexanderadam** [ruby-conferences/ruby-conferences.github.io#526]
Sometimes it would be nice to have some conference notes link.
Often I'm coming back from a conference and people ask me about highlights/gists/summaries and luckily some folks are doing notes (_i.e. [here](https://github.com/glowacki-dev/euruko-2022-notes/tree/main/talks), [here](https://github.com/glowacki-dev/euruko-2019-notes), [here](https://github.com/glowacki-dev/ruby-kaigi-notes) or [here](http://juixe.com/papers/RubyConf2006.pdf)_).

It would be nice if the ruby-conferences website would also be a place to have much (or even all) the information about a conference archived and linked.

Notes are often a useful part of that.

**Comment by adarsh** [comment]
Sounds like a good idea to me. How would you design this feature?

**Comment by marcoroth** [comment]
It might make more sense to host this on [rubyvideo.dev](https://rubyvideo.dev) now that we have that resource too!

**Comment by alexanderadam** [comment]
I agree. Then the past conferences should link to the according video recordings on rubyvideo though, right?
Should I create a separate issue for that and close this?

**Comment by marcoroth** [comment]
> Then the past conferences should link to the according video recordings on rubyvideo though, right?

Yes, that's my plan! 🙌🏼 

Feel free to keep this one open, maybe I can transfer the issue over, but we might want to open a new issue for updating the video links to point to rubyvideo.dev.
